certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -5115,9 +3678,61 @@
           },
           "404": {
             "x-nullable": false,
-            "description": "The doc or project could not be found",
+            "description": "The doc or project could not be found",
+            "schema": {
+              "$ref": "#/definitions/Microsoft.AspNetCore.Mvc.ProblemDetails"
+            }
+          }
+        }
+      }
+    },
+    "/api/Parameters/MultiExtract": {
+      "get": {
+        "tags": [
+          "Parameters"
+        ],
+        "summary": "Extract multiple document parameters",
+        "description": "Extract multiple document parameters",
+        "operationId": "Parameters_MultiExtract",
+        "produces": [
+          "text/plain",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "type": "array",
+            "name": "docids",
+            "in": "query",
+            "description": "The document id",
+            "collectionFormat": "multi",
```

Comment:
Please fix the below error. 
The 'collectionFormat' keyword value 'Multi' is not supported. ValuePath: /paths/~1api~1Parameters~1MultiExtract/get/parameters/0/collectionFormat